EapWebInterface/src/main/java/de/hpi/unicorn/application/pages/input/generator/attributeInput/FloatAttributeInput.java

```diff
@@ -0,0 +1,93 @@
+/*******************************************************************************
+ *
+ * Copyright (c) 2012-2017, Business Process Technology (BPT),
+ * http://bpt.hpi.uni-potsdam.de.
+ * All Rights Reserved.
+ *
+ *******************************************************************************/
+package de.hpi.unicorn.application.pages.input.generator.attributeInput;
+
+import de.hpi.unicorn.application.pages.input.generator.validation.RegexValidator;
+import de.hpi.unicorn.event.attribute.TypeTreeNode;
+import org.apache.commons.math3.distribution.NormalDistribution;
+import org.apache.commons.math3.distribution.UniformIntegerDistribution;
+import org.apache.log4j.Logger;
+import org.apache.wicket.validation.IValidator;
+
+import java.util.regex.Pattern;
+
+public class FloatAttributeInput extends AttributeInput {
+
+	private Float value;
+	{
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Move the contents of this initializer to a standard constructor or to field initializers. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AS1171)